pkg/virt-controller/watch/migration.go

```diff
@@ -219,8 +231,8 @@ func (md *MigrationDispatch) Execute(store cache.Store, queue workqueue.RateLimi
 				return
 			}
 			//TODO add a new state that more accurately reflects the process up the this point
-			//and then use MigrationInProgress to accurately indicate the actual migration pod is running
-			//setMigrationPhase(migration, v1.MigrationInProgress)
+			//and then use MigrationScheduled to accurately indicate the actual migration pod is running
```

Comment:
You were not introducing a new state here, so far you just renamed MigrationPending and MigrationInProgress.
At this point we have scheduled the pod which will do the migration, that is where we are missing the state transission.